kafka-rest/src/test/java/io/confluent/kafkarest/v2/KafkaConsumerManagerTest.java

```diff
@@ -70,6 +70,7 @@
 import io.confluent.kafkarest.entities.EmbeddedFormat;
 import io.confluent.kafkarest.entities.TopicPartitionOffset;
 import io.confluent.rest.RestConfigException;
+import org.junit.jupiter.api.Disabled;
```

Comment:
could you remove this import as this branch is with Junit4 so it would be import for Ignore